Sources/MessagesCollectionViewFlowLayout.swift

```diff
@@ -30,6 +30,8 @@ open class MessagesCollectionViewFlowLayout: UICollectionViewFlowLayout {
     // MARK: - Properties
 
     open var messageLabelFont: UIFont
+    open var emojiLabelFont: UIFont
```

Comment:
I think this should be private if we're going to override it by doubling messageLabelFont